spec/helpers/format_helper_spec.rb

```diff
@@ -10,10 +10,6 @@
     end
 
     it 'should return HTML for header markdown' do
-      expect(Redcarpet::Markdown).to receive(:new)
-      .with(Redcarpet::Render::HTML, autolink: true, space_after_headers: true, no_intra_emphasis: true, fenced_code_blocks: true, disable_indented_code_blocks: true, safe_links_only: true)
-      .and_call_original
```

Comment:
Hoping it’s okay to remove this from the test; it seems excessive for such a simple method.